README.md

```diff
@@ -263,3 +263,6 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
 
 ## Data mining
   * http://www.monkeylearn.com/ - Text mining in the cloud, 1,000 queries for free per month.
+
+## API
```

Comment:
dont really want to add such a niche category, i think this can go in the platforms section